### DISCUSSION TITLE
Welcome to phpcollab Discussions!

### DISCUSSION THREAD

**Discussion by mindblender** [phpcollab/phpcollab#62] [General]
Greetings!

Welcome to [GitHub Discussion](https://docs.github.com/pt/discussions). This is a place to communicate with the community and team behind [phpCollab](https://phpcollab.com). If you have a question about how to use the app, wonder why it takes so long to get changes released 😂, or want to request a new feature, then just start a new discussion!

**Comment by Machiuka** [comment]
Hello,

I try to use phpcollab. But it seems is a demo version. I defined users for a client, projects for a client, but when I want to assign team members to a project it didn;t allow me to do that. Only demo user is available. This is a limitation of the demo version?

**Comment by nhoknakroz** [comment]
Hi @mindblender,

I try to reset password but it always return "You have tried too many times. Please wait and try again.". Did I do something wrong?

**Reply by mindblender** [reply]
@Machiuka 
Hmm ok.  I will do some more testing on that functionality.

Could you please open an issue for this? Thanks!

**Comment by mumonkey** [comment]
I was tasked (pun intended) with finding a server based PM software for our company. We make widgets for other Clients. They send is CAD files. We give them quotes. We order parts. We make many parts to. We weld parts together and assemble all those parts into a widget. I have to track a lot of moving parts. 
Our boss didn't want a complex thing to learn. The Gnatt chart on the same page as the project is what sold us all. 
We have 3 clients and 30+ suppliers/vendors. We suggest the following:
1. vendor DB
2. $per hour (you have Est time and hours worked. Link the hourly rate to the hours worked to see what the job cost vs budgeted)
3. Parts DB from vendors. I order the same parts over and over. 
4. Bulk importing (I just completed manually copying 102 tasks from MS projects) We have so many more
5. Copy Similar project. ( I don't want to do all that each time we make the same widget)
6. Ability to edit or add "terms" to drop down menus. (status: Bug Found)
7. Ticket: Client or employee can open a ticket for common things like reporting bugs,defects, reworks, change orders. 
8. Can we put our company name in the upper left? 
Contact me to see what I've done on my site. I'm on Pacific Time. If this works my employer said he will donate.

**Reply by mindblender** [reply]
Hi @mumonkey Thanks for the great feedback and suggestions.  

I have some questions about the things you are asking, so I'll get down to it.

`1.  vendor DB`
Are you wanting a field on the page with a list of vendors, or something else?

`2. $per hour (you have Est time and hours worked. Link the hourly rate to the hours worked to see what the job cost vs budgeted)`
Could you please expand on this?  Not sure what you are wanting changed since this is already in the invoicing piece.

`3. Parts DB from vendors. I order the same parts over and over.`
Not sure how this one would be handled.  Who would maintain the DB, where would it be integrated?

`4. Bulk importing (I just completed manually copying 102 tasks from MS projects) We have so many more`
Bulk importing of tasks/data?  Currently phpCollab does not have any import functionality but it might be possible to add that functionality.  I would need to know more about the use case to determine if this is something beneficial to the greater community or if this is a one-off customization.

`5. Copy Similar project. ( I don't want to do all that each time we make the same widget)`
The "copy a project" functionality should already be there.  You can also setup phases that can be re-used if you follow certain paths for handling client projects.  If this is not what you are wanting, or if it does not do what you are expecting, let me know and I'd be happy to look into it.

`6. Ability to edit or add "terms" to drop down menus. (status: Bug Found)`
The status drop down can be customized by editing the appropriate language file. See https://github.com/phpcollab/phpcollab/blob/master/languages/lang_en.php#L34 Right now this is a manual process, edit the code file, but perhaps in a future version this could be added to the Admin UI to make it easier.

`7. Ticket: Client or employee can open a ticket for common things like reporting bugs,defects, reworks, change orders.`
From the "project site/client side" a support ticket can be created.  This can also be viewed on the Projects view.

`8. Can we put our company name in the upper left?`
Yes, on the main site, the company name should appear if there is no logo.  This is set in the settings.php file

I would be open to discussing these further, and to see what you have done on your site.

**Comment by mumonkey** [comment]
If I click any sorting of tasks. I get the ooof error.  We enter tasks in the order we need them to start. We need to view them in the same order of the next task. The default sort is alphabetical. 

I'm a Mac guy and he is antimac. He wants to buy a computer so I can use MS projects. I don't even think they make it anymore. I don't expect you to be able to make any changes. My boss likes simple then.... you know what I mean. If there was software that had the ability to turn on/off whatever bells and whistles we want. 


see answers below

**Reply by mindblender** [reply]
I understand.

As for the sorting, there is a bug, which I have fixed and it will be in the next release.

If you are comfortable with modifying files, you can pull down the fixed file and manually patch it. See this [commit](https://github.com/phpcollab/phpcollab/commit/638338fe22dad481677e5981f3574b6116f98514)

Feel free to join [Slack](https://slack.phpcollab.com/) and ping me there and we can discuss what you are looking for.

**Comment by mumonkey** [comment]
Do you want to see what I’ve done?

***@***.***

**Reply by mindblender** [reply]
@mumonkey I hope that is not your production site, since this is a public forum and anyone could gain access to it.  I'd suggest resetting the admin password :)

**Reply by mumonkey** [reply]
I was replying to an email. I didn't know it was posted.. Thanks for the warning

**Comment by nhoknakroz** [comment]
Hi @mindblender, I saw that there are some changes since the latest version of phpCollab. Can you release new version with these changes?

**Comment by mindblender** [comment]
Hi @nhoknakroz 
I do have plans on doing a release as soon as I address another issue.  🤞 it will be soon.
